pkgs/top-level/all-packages.nix

```diff
@@ -22704,7 +22704,6 @@ in
   solarc-gtk-theme = callPackage ../misc/themes/solarc { };
 
   xfce = xfce4-12;
```

Comment:
Should we set `xfce = xfce4-14` ?

Reply:
Yeah, but I plan to do that for 20.03 and I'd like to backport this PR.

Reply:
Yes, but "for 20.03" is "in current master" because 19.09 is already forked, right?